lib/services/devices/deviceService.js

```diff
@@ -363,6 +363,7 @@ function registerDevice(deviceObj, callback) {
                     deviceObj.type = deviceData.type;
                     deviceObj.staticAttributes = deviceData.staticAttributes;
                     deviceObj.commands = deviceData.commands;
+                    deviceObj.lazy = deviceData.lazy;
```

Comment:
Could you explain why you included this line and how it is related with functionalities implemented in this PR (merge patch)

Reply:
This is necessary to ensure that a lazy attribute may be merged-patched (see use on line lib/services/northBound/contextServer-NGSI-LD.js - 455)